[32mINFO    [0m logger:test_api.py:521 test delete contact successfully complete